wp-content/themes/reconasia/assets/_scss/layout/_footer.scss

```diff
@@ -60,31 +72,36 @@
   &__logo {
     grid-area: logo;
     max-width: 300px;
+  }
 
-    &,
-    &:visited {
-      // color: $color-text-gray-10;
-    }
+  &__desc {
+    grid-area: csisdesc;
   }
 
   .widget:first-of-type {
-    grid-area: csisdesc;
+    margin-bottom: rem(24);
   }
 
   .widget:nth-of-type(2) {
-    grid-area: magdesc;
+    margin-bottom: rem(32);
+  }
+
+  .widget:nth-of-type(3) {
+    grid-area: newsletter;
+    max-width: rem(252);
   }
```

Comment:
I don't see where this is effecting any elements.  Since there are a lot of elements with the class of `widget` and you're trying to target the element directly after the description, I would use `&__desc + .widget` to target this element.

Also, why is there a `max-width` on this?